### PR TITLE
TableGraph base functionality

### PR DESCRIPTION
**Pull request by ischolten**
- [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergable
  - [x] Tests pass

Connect #1686 

### The problem
There is no table graph.
### The Solution
Now there is a basic table graph. 

The feature is "turned off". To "turn on" the feature, uncomment lines 506-511 in dashboards/graphics/graph.js

### CODE REVIEW

File: ui/src/shared/components/TableGraph.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import {MultiGrid} from 'react-virtualized/dist/commonjs/MultiGrid'
+
+class TableGraph extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+  componentWillMount() {
+    this._labels = []
+    this._data = [[]]
```
Comment (lukevmorris) [inline ui/src/shared/components/TableGraph.js]: Is there a reason for phrasing these as instance variables as opposed to component state? These don't have the benefit of an automatic component rerender if they change

File: ui/src/shared/components/TableGraph.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    const tableWidth = this.gridContainer ? this.gridContainer.clientWidth : 0
+    const tableHeight = this.gridContainer ? this.gridContainer.clientHeight : 0
+
+    return (
+      <div
+        className="graph-container"
+        ref={gridContainer => (this.gridContainer = gridContainer)}
+      >
+        {data.length > 1
+          ? <MultiGrid
```
Comment (lukevmorris) [inline ui/src/shared/components/TableGraph.js]: I think in general we should be using `&&` if the ternary else clause is `null`

File: ui/src/shared/components/TableGraph.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React, {PropTypes, Component} from 'react'
+import _ from 'lodash'
+import {timeSeriesToTable} from 'src/utils/timeSeriesToDygraph'
+import {MultiGrid} from 'react-virtualized/dist/commonjs/MultiGrid'
+
+class TableGraph extends Component {
+  constructor(props) {
+    super(props)
+  }
```
[the remark this answers is not in the document]
Comment (ebb-tide) [reply ui/src/shared/components/TableGraph.js]: hrmmm.. if only this suggestion were in the style guide.....

File: ui/src/shared/components/TableGraph.js (hunk 2 above)
Comment (ebb-tide) [reply ui/src/shared/components/TableGraph.js]: style guide or it didn't happen.

File: ui/src/shared/components/TableGraph.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import React, {PropTypes, Component} from 'react'
+import _ from 'lodash'
+import {timeSeriesToTable} from 'src/utils/timeSeriesToDygraph'
+import {MultiGrid} from 'react-virtualized/dist/commonjs/MultiGrid'
```
[the remark this answers is not in the document]
Comment (ebb-tide) [reply ui/src/shared/components/TableGraph.js]: vscode is weird.

File: ui/src/dashboards/components/GraphOptionsCustomizableColumn.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+import InputClickToEdit from 'shared/components/InputClickToEdit'
+
+const GraphOptionsCustomizableColumn = ({
+  originalColumnName,
+  newColumnName,
+  onColumnRename,
+}) => {
+  return (
+    <div className="gauge-controls--section">
```
[the remark this answers is not in the document]
Comment (ebb-tide) [reply ui/src/dashboards/components/GraphOptionsCustomizableColumn.js]: I have left this as TODO items- refactoring is underway.

File: ui/src/shared/components/TableGraph.js (hunk 2 above)
Comment (lukevmorris) [reply ui/src/shared/components/TableGraph.js]: adding

File: ui/src/shared/components/TableGraph.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (lukevmorris) [reply ui/src/shared/components/TableGraph.js]: adding

**Pull request merged**